irohad/model/execution/impl/command_executor.cpp

```diff
@@ -511,24 +500,22 @@ namespace iroha {
       account.json_data = "{}";
       auto domain = queries.getDomain(create_account.domain_id);
       if (not domain.has_value()) {
-        return makeExecutionError(
+        return makeExecutionResult(
             (boost::format("Domain %s not found") % create_account.domain_id)
                 .str());
       }
       // TODO: remove insert signatory from here ?
```

Comment:
Pls, fix the todo style and write a task for it.

Reply:
Okay

Reply:
https://github.com/hyperledger/iroha/pull/941